git-interactive-rebase-tool.1

```diff
@@ -0,0 +1,26 @@
+.\" Manpage for interactive-rebase-tool.
+.\" Contact dev@mitmaro.ca to correct errors or typos.
+.TH INTERACTIVE\-REBAS 1 "02/02/2018" "v0.5.0" "Git Manual"
+.SH NAME
+git-interactive-rebase-tool \- an improved sequence editor for git
+.SH SYNOPSIS
+.nf
+\fIgit interactive-rebase-tool\fR <rebase-todo-filepath>
+\fIgit interactive-rebase-tool\fR --version | -v
```

Comment:
Remove `git ` from these lines. Since the tool isn't installed with the `git` prefix it can't be run with git.